admin/sqlpatch.php

```diff
@@ -282,7 +283,6 @@ function executeSql($lines, $database, $table_prefix = '') {
           default:
             break;
           } //end switch
-//        } // endif $table_prefix

```

Comment:
Your bracket-matching comment is understood. But you've removed only the 2nd half of the bracket ... so you're actually creating a bracket-matching problem by not removing the opening bracket up at 113/114.